internals/docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:6-slim
+
+ENV KARMA_BROWSER PhantomJS
+RUN apt-get update && apt-get install libpng12-0
+
+WORKDIR /reactapp
```

Comment:
It set the working directory for docker command onward.
`COPY ~/HostDirectory .`
will copy file from `~/HostDirectory` to `/reactapp`